reco_utils/recommender/tfidf/tfidf_utils.py

```diff
@@ -114,18 +114,18 @@ def clean_dataframe(self, df, cols_to_clean, new_col_name="cleaned_text"):
     def tokenize_text(
         self, df_clean, text_col="cleaned_text", ngram_range=(1, 3), min_df=0
     ):
-        """ Tokenize the input text.
-            For more details on the TfidfVectorizer, see https://scikit-learn.org/stable/modules/generated/sklearn.feature_extraction.text.TfidfVectorizer.html
-        
+        """Tokenize the input text.
+        For more details on the TfidfVectorizer, see https://scikit-learn.org/stable/modules/generated/sklearn.feature_extraction.text.TfidfVectorizer.html
+
         Args:
             df_clean (pandas.DataFrame): Dataframe with cleaned text in the new column.
             text_col (str): Name of column containing the cleaned text.
             ngram_range (tuple of int): The lower and upper boundary of the range of n-values for different n-grams to be extracted.
             min_df (int): When building the vocabulary ignore terms that have a document frequency strictly lower than the given threshold.
-        
+
         Returns:
-            TfidfVectorizer, pd.Series: 
-            - Scikit-learn TfidfVectorizer object defined in .tokenize_text().
+            TfidfVectorizer, pd.Series:
```

Comment:
This should be `pandas`

Reply:
And also `sklearn.feature_extraction.text.TfidfVectorizer`

Reply:
addressed